app/app.py

```diff
@@ -24,7 +25,7 @@ def question_router(question_type, question_id):
 
 @app.post(API_VERSION + "/question/<question_type>/<question_id>")
 def get_solution(question_type, question_id):
-    attempt = request.get_json()
+    attempt = request.get_json(force=True)
```

Comment:
Added this back in?

Reply:
yeah -_- I'll need to do more research on this, just put this here to continue getting everything else working